InstallSpinnaker.sh

```diff
@@ -50,8 +50,8 @@ fi
 
 function print_usage() {
   cat <<EOF
-usage: $0 [--cloud_provider <aws|google|none|both>]
-    [--aws_region <region>] [--google_region <region>]
+usage: $0 [--cloud_provider <aws|google|azure|none|both|all>]
```

Comment:
we should deprecate "both".
I'm not sure "all" is going to make sense in practice, but I guess would not hurt.